pubspec.yaml

```diff
@@ -12,6 +12,8 @@ dependencies:
   json_annotation: ^4.6.0
   web: '>=0.4.0 <1.0.0'
   js: '>=0.6.0 <1.0.0'
+  flutter_bloc: ^8.0.1
```

Comment:
we'll need to specify a range for any added dependencies so that downstream apps can differ on which versions they use. i would see how far back you can go before our code becomes in compatible, then set a `>=x.x.x <9.0.0` here